careers/technical-engineer.md

```diff
@@ -16,63 +16,67 @@ The senior engineer displays a mastery of the core competencies of software engi
 
 **Collaboration:** They ensure that they do not become a single point of failure for their domain by mentoring and training others. When relevant, they work effectively with others outside the organization (whether in other departments or with vendors). They are capable of effectively running projects of any size independently.
 
+### Responsibilities
+
+* Provide technical leadership and guidance across the product and engineering organizations.
+* Make key architectural decisions and ensure the scalability, performance, and reliability of critical systems.
+* Mentor and coach junior and mid-level engineers to foster their technical growth.
+* Collaborate with product managers, designers, and stakeholders to define requirements and solutions.
+* Drive the technical vision and strategy for your area of responsibility, aligning with organizational goals.
+* Conduct thorough code reviews to continually improve the overall quality of our codebase.
```

Comment:
I really like this bit from "Communication" above: 

> When pairing with more junior team members, they guide the conversation by asking questions instead of just providing answers. 

I think we can integrate that here:

> Elevate code quality through thoughtful reviews that educate and guide through questioning rather than dictating, helping others develop sound technical judgment.